js/main.js

```diff
@@ -77,12 +79,8 @@ function renderAllEntries(event) {
   viewSwap(data.view);
 }
 
-function toggleNoEntries(entriesCount) {
-  if (entriesCount === 0) {
-    $noEntries.classList.remove('hidden');
-  } else {
-    $noEntries.classList.add('hidden');
-  }
+function toggleNoEntries() {
+  $noEntries.classList.add('hidden');
```

Comment:
One small change here is to use `toggle` instead of `add`. Toggle will check if the classList contains 'hidden', if yes it will remove, and if no it will add it. This is just for forward planning to help you when you get to future feature.